src/main/java/net/imglib2/img/array/ArrayImgs.java

```diff
@@ -353,6 +370,111 @@ private ArrayImgs()
 	}
 
 	/**
+	 * Create an {@link ArrayImg}&lt;{@link Unsigned2BitType}, {@link LongArray}&gt;.
+	 */
+	@SuppressWarnings( "unchecked" )
+	final static public ArrayImg< Unsigned2BitType, LongArray > unsigned2Bits( final long... dim )
+	{
+		return ( ArrayImg< Unsigned2BitType, LongArray > ) new ArrayImgFactory< Unsigned2BitType >().create( dim, new Unsigned2BitType() );
+	}
+
+	/**
+	 * Creates an {@link ArrayImg}&lt;{@link Unsigned2BitType}, {@link LongAccess}&gt;
+	 * using a {@link LongAccess} passed as argument.
+	 */
+	final static public < A extends LongAccess > ArrayImg< Unsigned2BitType, A > unsigned2Bits( final A access, final long... dim )
+	{
+		final ArrayImg< Unsigned2BitType, A > img = new ArrayImg<>( access, dim, new Fraction( 1, 64 ) );
```

Comment:
Shouldn't this be `new Fraction( t.getBitsPerPixel(), 64 )` instead of `new Fraction( 1, 64 )` here?

Reply:
Well, we are able to hard code in a 2 here instead of `t.getBitsPerPixel()`, but yes, you are right.